corgidrp/mocks.py

```diff
@@ -261,6 +264,160 @@ def create_simflat_dataset(filedir=None, numfiles=10):
     dataset = data.Dataset(frames)
     return dataset
 
+
+def create_raster(mask,data,dither_sizex=None,dither_sizey=None,row_cent = None,col_cent = None,n_dith=1,mask_size=180,snr=250,planims=None):
+    """Performs raster scan of Neptune or Uranus images
+    
+     Args:
+        mask (int): (Required)  Mask used for the image.
+        data (float):(Required) Data in array npixels*npixels format to be raster scanned
+        dither_sizex (int):(Required) Size of the dither in X axis in pixels
+        dither_sizey (int):(Required) Size of the dither in X axis in pixels
+        row_cent (int): (Required)  X coordinate of the centroid
+        col_cent (int): (Required)  Y coordinate of the centroid
+        n_dith (int): number of dithers required
+        mask_size (int): Size of the mask in pixels
+        snr (int): Required SNR in the planet images
+        planims (str): Planet and band
```

Comment:
please provide the valid options in the docstring, given there are only 4 valid options